eos/constraint.cc

```diff
@@ -1145,19 +1146,30 @@ namespace eos
                     throw InternalError("make_multivariate_gaussian_covariance_constraint<measurements=" + stringify(dim_meas) + ",predictions=" + stringify(dim_pred) + ">: " + name.str() + ": '" + this->observables[i].str() + "' is not a valid observable name");
             }
 
-            // create GSL vector for the mean
-            gsl_vector * means = gsl_vector_alloc(dim_meas);
-            gsl_vector_memcpy(means, this->means);
+	    //If specified, these options allow to specify a subset of the measurements
+	    unsigned begin = destringify<unsigned>(options.get("begin", "0"));
+	    unsigned end = destringify<unsigned>(options.get("end", stringify(dim_meas)));
+	    unsigned sub_dim = end - begin;
```

Comment:
I was wondering if it would be helpful (for potential later adaptations) to name this ```subdim_meas```?

Reply:
Absolutely, the name sub_dim was referring to "subset dimension", but yours is better